diff --git a/node_modules/@babel/types/LICENSE b/node_modules/@babel/types/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/README.md b/node_modules/@babel/types/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/asserts/assertNode.js b/node_modules/@babel/types/lib/asserts/assertNode.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/asserts/generated/index.js b/node_modules/@babel/types/lib/asserts/generated/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/builders/builder.js b/node_modules/@babel/types/lib/builders/builder.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/builders/flow/createFlowUnionType.js b/node_modules/@babel/types/lib/builders/flow/createFlowUnionType.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/builders/flow/createTypeAnnotationBasedOnTypeof.js b/node_modules/@babel/types/lib/builders/flow/createTypeAnnotationBasedOnTypeof.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/builders/generated/index.js b/node_modules/@babel/types/lib/builders/generated/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/builders/react/buildChildren.js b/node_modules/@babel/types/lib/builders/react/buildChildren.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/builders/typescript/createTSUnionType.js b/node_modules/@babel/types/lib/builders/typescript/createTSUnionType.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/clone/clone.js b/node_modules/@babel/types/lib/clone/clone.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/clone/cloneDeep.js b/node_modules/@babel/types/lib/clone/cloneDeep.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/clone/cloneDeepWithoutLoc.js b/node_modules/@babel/types/lib/clone/cloneDeepWithoutLoc.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/clone/cloneNode.js b/node_modules/@babel/types/lib/clone/cloneNode.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/clone/cloneWithoutLoc.js b/node_modules/@babel/types/lib/clone/cloneWithoutLoc.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/comments/addComment.js b/node_modules/@babel/types/lib/comments/addComment.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/comments/addComments.js b/node_modules/@babel/types/lib/comments/addComments.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/comments/inheritInnerComments.js b/node_modules/@babel/types/lib/comments/inheritInnerComments.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/comments/inheritLeadingComments.js b/node_modules/@babel/types/lib/comments/inheritLeadingComments.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/comments/inheritTrailingComments.js b/node_modules/@babel/types/lib/comments/inheritTrailingComments.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/comments/inheritsComments.js b/node_modules/@babel/types/lib/comments/inheritsComments.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/comments/removeComments.js b/node_modules/@babel/types/lib/comments/removeComments.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/constants/generated/index.js b/node_modules/@babel/types/lib/constants/generated/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/constants/index.js b/node_modules/@babel/types/lib/constants/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/converters/ensureBlock.js b/node_modules/@babel/types/lib/converters/ensureBlock.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/converters/gatherSequenceExpressions.js b/node_modules/@babel/types/lib/converters/gatherSequenceExpressions.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/converters/toBindingIdentifierName.js b/node_modules/@babel/types/lib/converters/toBindingIdentifierName.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/converters/toBlock.js b/node_modules/@babel/types/lib/converters/toBlock.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/converters/toComputedKey.js b/node_modules/@babel/types/lib/converters/toComputedKey.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/converters/toExpression.js b/node_modules/@babel/types/lib/converters/toExpression.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/converters/toIdentifier.js b/node_modules/@babel/types/lib/converters/toIdentifier.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/converters/toKeyAlias.js b/node_modules/@babel/types/lib/converters/toKeyAlias.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/converters/toSequenceExpression.js b/node_modules/@babel/types/lib/converters/toSequenceExpression.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/converters/toStatement.js b/node_modules/@babel/types/lib/converters/toStatement.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/converters/valueToNode.js b/node_modules/@babel/types/lib/converters/valueToNode.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/definitions/core.js b/node_modules/@babel/types/lib/definitions/core.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/definitions/es2015.js b/node_modules/@babel/types/lib/definitions/es2015.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/definitions/experimental.js b/node_modules/@babel/types/lib/definitions/experimental.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/definitions/flow.js b/node_modules/@babel/types/lib/definitions/flow.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/definitions/index.js b/node_modules/@babel/types/lib/definitions/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/definitions/jsx.js b/node_modules/@babel/types/lib/definitions/jsx.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/definitions/misc.js b/node_modules/@babel/types/lib/definitions/misc.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/definitions/placeholders.js b/node_modules/@babel/types/lib/definitions/placeholders.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/definitions/typescript.js b/node_modules/@babel/types/lib/definitions/typescript.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/definitions/utils.js b/node_modules/@babel/types/lib/definitions/utils.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/index.d.ts b/node_modules/@babel/types/lib/index.d.ts
old mode 100644
new mode 100755
index 0f14e9e..11c879b
--- a/node_modules/@babel/types/lib/index.d.ts
+++ b/node_modules/@babel/types/lib/index.d.ts
@@ -2173,7 +2173,7 @@ export function isNullLiteralTypeAnnotation(node: object | null | undefined, opt
 // export function assertNullLiteralTypeAnnotation(node: object | null | undefined, opts?: object | null): asserts node is NullLiteralTypeAnnotation;
 export function isNullableTypeAnnotation(node: object | null | undefined, opts?: object | null): node is NullableTypeAnnotation;
 // export function assertNullableTypeAnnotation(node: object | null | undefined, opts?: object | null): asserts node is NullableTypeAnnotation;
-export function isNumberLiteral(node: object | null | undefined, opts?: object | null): boolean;
+export function isNumberLiteral(node: object | null | undefined, opts?: object | null): node is NumericLiteral;
 // export function assertNumberLiteral(node: object | null | undefined, opts?: object | null): asserts node;
 export function isNumberLiteralTypeAnnotation(node: object | null | undefined, opts?: object | null): node is NumberLiteralTypeAnnotation;
 // export function assertNumberLiteralTypeAnnotation(node: object | null | undefined, opts?: object | null): asserts node is NumberLiteralTypeAnnotation;
diff --git a/node_modules/@babel/types/lib/index.js b/node_modules/@babel/types/lib/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/index.js.flow b/node_modules/@babel/types/lib/index.js.flow
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/modifications/appendToMemberExpression.js b/node_modules/@babel/types/lib/modifications/appendToMemberExpression.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/modifications/flow/removeTypeDuplicates.js b/node_modules/@babel/types/lib/modifications/flow/removeTypeDuplicates.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/modifications/inherits.js b/node_modules/@babel/types/lib/modifications/inherits.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/modifications/prependToMemberExpression.js b/node_modules/@babel/types/lib/modifications/prependToMemberExpression.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/modifications/removeProperties.js b/node_modules/@babel/types/lib/modifications/removeProperties.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/modifications/removePropertiesDeep.js b/node_modules/@babel/types/lib/modifications/removePropertiesDeep.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/modifications/typescript/removeTypeDuplicates.js b/node_modules/@babel/types/lib/modifications/typescript/removeTypeDuplicates.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/retrievers/getBindingIdentifiers.js b/node_modules/@babel/types/lib/retrievers/getBindingIdentifiers.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/retrievers/getOuterBindingIdentifiers.js b/node_modules/@babel/types/lib/retrievers/getOuterBindingIdentifiers.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/traverse/traverse.js b/node_modules/@babel/types/lib/traverse/traverse.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/traverse/traverseFast.js b/node_modules/@babel/types/lib/traverse/traverseFast.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/utils/inherit.js b/node_modules/@babel/types/lib/utils/inherit.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/utils/react/cleanJSXElementLiteralChild.js b/node_modules/@babel/types/lib/utils/react/cleanJSXElementLiteralChild.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/utils/shallowEqual.js b/node_modules/@babel/types/lib/utils/shallowEqual.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/buildMatchMemberExpression.js b/node_modules/@babel/types/lib/validators/buildMatchMemberExpression.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/generated/index.js b/node_modules/@babel/types/lib/validators/generated/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/is.js b/node_modules/@babel/types/lib/validators/is.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isBinding.js b/node_modules/@babel/types/lib/validators/isBinding.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isBlockScoped.js b/node_modules/@babel/types/lib/validators/isBlockScoped.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isImmutable.js b/node_modules/@babel/types/lib/validators/isImmutable.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isLet.js b/node_modules/@babel/types/lib/validators/isLet.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isNode.js b/node_modules/@babel/types/lib/validators/isNode.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isNodesEquivalent.js b/node_modules/@babel/types/lib/validators/isNodesEquivalent.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isPlaceholderType.js b/node_modules/@babel/types/lib/validators/isPlaceholderType.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isReferenced.js b/node_modules/@babel/types/lib/validators/isReferenced.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isScope.js b/node_modules/@babel/types/lib/validators/isScope.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isSpecifierDefault.js b/node_modules/@babel/types/lib/validators/isSpecifierDefault.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isType.js b/node_modules/@babel/types/lib/validators/isType.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isValidES3Identifier.js b/node_modules/@babel/types/lib/validators/isValidES3Identifier.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isValidIdentifier.js b/node_modules/@babel/types/lib/validators/isValidIdentifier.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/isVar.js b/node_modules/@babel/types/lib/validators/isVar.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/matchesPattern.js b/node_modules/@babel/types/lib/validators/matchesPattern.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/react/isCompatTag.js b/node_modules/@babel/types/lib/validators/react/isCompatTag.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/react/isReactComponent.js b/node_modules/@babel/types/lib/validators/react/isReactComponent.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/lib/validators/validate.js b/node_modules/@babel/types/lib/validators/validate.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/scripts/generateTypeHelpers.js b/node_modules/@babel/types/scripts/generateTypeHelpers.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/scripts/generators/docs.js b/node_modules/@babel/types/scripts/generators/docs.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/scripts/generators/flow.js b/node_modules/@babel/types/scripts/generators/flow.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/scripts/generators/generateAsserts.js b/node_modules/@babel/types/scripts/generators/generateAsserts.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/scripts/generators/generateBuilders.js b/node_modules/@babel/types/scripts/generators/generateBuilders.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/scripts/generators/generateConstants.js b/node_modules/@babel/types/scripts/generators/generateConstants.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/scripts/generators/generateValidators.js b/node_modules/@babel/types/scripts/generators/generateValidators.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/scripts/generators/typescript.js b/node_modules/@babel/types/scripts/generators/typescript.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/scripts/utils/formatBuilderName.js b/node_modules/@babel/types/scripts/utils/formatBuilderName.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/scripts/utils/lowerFirst.js b/node_modules/@babel/types/scripts/utils/lowerFirst.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/scripts/utils/stringifyValidator.js b/node_modules/@babel/types/scripts/utils/stringifyValidator.js
old mode 100644
new mode 100755
diff --git a/node_modules/@babel/types/scripts/utils/toFunctionName.js b/node_modules/@babel/types/scripts/utils/toFunctionName.js
old mode 100644
new mode 100755
